microcosm_metrics/naming.py

```diff
@@ -0,0 +1,8 @@
+"""
+Naming conventions.
+
+"""
+
+
+def name_for(key, prefix="microcosm"):
```

Comment:
We need a way to differentiate our metrics from others generated by other systems. So far, I'm just prefixing, but we'll see how it develops.